taworks/taform/templates/taform/algorithm.html

```diff
@@ -31,29 +31,56 @@ <h1>Teaching Assistant Assignment</h1>
         <td>
         <form action="{% url 'algorithm' %}" method="post">
         {% csrf_token %}
-            <input type="submit" name="algo_run" value="Go!"/>
+            <input class="submitButton2" type="submit" name="algo_run" value="Solve model"/>
         </form>
         </td>
     </tr>
     <tr>
-        <td>2. Export algorithm results.&emsp;</td>
+        <td>2. Export students without a position.&emsp;</td>
         <td>
         <form action="{% url 'algorithm' %}" method="post">
         {% csrf_token %}
-            <input type="submit" name="algo_export" value="Export"/>
+            <input class="submitButton2" type="submit" name="student_export" value="Export students w/o match"/>
```

Comment:
Can we make the buttons say "Students without a match" and "Courses without a match"?